.github/workflows/main.yml

```diff
@@ -0,0 +1,44 @@
+name: Main CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ~/vscode-camel-extension-pack
```

Comment:
is it needed on GitHub Actions?

Reply:
not necessarily.. if we are not requiring using the ~ as root folder than it is not needed